YAMS-LIB/Program.cs

```diff
@@ -2282,6 +2296,9 @@ void RotateTextureAndSaveToTexturePage(int rotation, UndertaleTexturePageItem te
                 case ItemEnum.Flashlight:
                     ReplaceGMLInCode(characterVarsCode, "global.flashlightLevel = 0", $"global.flashlightLevel = {quantity};");
                     break;
+                case ItemEnum.Blindfold:
+                    ReplaceGMLInCode(characterVarsCode, "global.flashlightLevel = 0", $"global.flashlightLevel = {quantity};");
```

Comment:
this should be `= -{quantity}`. since if we start with 2 blindfolds, we want a negative level.